src/world.rs

```diff
@@ -286,6 +282,8 @@ impl World {
 
     /// Get an immutable reference to the component of the given type for an [Entity].
     ///
+    /// Note that [QueryMap](struct.QueryMap.html) provides faster random access to components.
```

Comment:
For some reason, it was not possible to use the short link form `[QueryMap]` here. Not really clear to me why.

Reply:
The links use the same scope as the compiler would, i.e. you can only use `QueryMap` if you `use crate::query::QueryMap` here. In this case you could link `crate::QueryMap` since it is re-exported.

Reply:
But since I would prefer the example with `QueryRef::map`, I would also suggest to link this here.

Reply:
Same thing there, of course. Changed the link, but would like to avoid the hard to read `crate::QueryRef::map()` to the readers of the docs.

Reply:
You can always do `[QueryRef::map](crate::QueryRef::map)` to avoid that. Will amend after merging...